hudi-io/src/main/java/org/apache/hudi/common/util/StringUtils.java

```diff
@@ -33,8 +33,10 @@
  */
 public class StringUtils {
 
-  public static final char[] HEX_CHAR = new char[]{'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+  public static final char[] HEX_CHAR = new char[] {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
```

Comment:
nit: necessary space?

Reply:
It's changed by IDE by following the code style.